src/main/java/ch/njol/skript/lang/Variable.java

```diff
@@ -119,7 +125,26 @@ private Variable(VariableString name, Class<? extends T>[] types, boolean local,
 	 * @return true if the name is valid, false otherwise.
 	 */
 	public static boolean isValidVariableName(String name, boolean allowListVariable, boolean printErrors) {
-		name = name.startsWith(LOCAL_VARIABLE_TOKEN) ? "" + name.substring(LOCAL_VARIABLE_TOKEN.length()).trim() : "" + name.trim();
+		assert !name.isEmpty(): "Variable name should not be empty";
+		char first = name.charAt(0);
+		check_reserved_tokens:
+		for (char token : reservedTokens) {
```

Comment:
```suggestion
		check_reserved_tokens: for (char token : reservedTokens) {
```
I feel like it's clearer to typically combine these, but I'm not sure what the code conventions suggest (if anything)

Reply:
Reformat puts it on a new line so I usually put it on a new line, up to you though.

Reply:
It's fine as is if it's just the reformatter